src/RealtimeClient.ts

```diff
@@ -102,6 +102,10 @@ export default class RealtimeClient {
     if (eventsPerSecond)
       this.eventsPerSecondLimitMs = Math.floor(1000 / eventsPerSecond)
 
+    const accessToken = options?.params?.apikey
+    if (accessToken) this.accessToken = accessToken
+    console.log({ accessToken })
```

Comment:
@dshukertjr did you intend to keep this in there?

Reply:
Oops, no didn't mean to include the console log. Sorry for that.